src/elements/element.rectangle.js

```diff
@@ -132,8 +142,14 @@ module.exports = Element.extend({
 		// Draw rectangle from 'startCorner'
 		var corner = cornerAt(0);
 		ctx.moveTo(corner[0], corner[1]);
+		//	float-bar support, let's rectangle allow to have all borders, assign corners_count to 5 instead of 4, so we are allowing to fill all sides.
+		var cornersCount = 4;
```

Comment:
it might be nice to use ternary operator here:

    var cornersCount = borderSkipped === null ? 5 : 4;

Reply:
And actually we should probably call it `edgeCount` instead of cornersCount. We never have 5 corners

    var edgeCount = borderSkipped === null ? 4 : 3;
    for (var i = 1; i <= edgeCount; i++) {